HTML-CSS/grid-proje/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+    <div class="container">
+        <div class="left-side">
+            <div class="red-1"></div>
+            <div class="white-1"></div>
+        </div>
+        <div class="right-side">
+            <div class="row-1">
+                <div class="col-1"></div>
```

Comment:
row-1 col-1 yerine daha box yapısına uygun isimler verebilirsin. grid sistemi olustururken postfix olarak verdiğimiz numeric ifadeler aslında oransal olarak o content'in ne kadar alan kapladığını ifade ediyor. style kodlarından anladığım kadarıyla sen her row ve her rowa ait col altında farklı grid sistemleri kurduğun için daha öznel isimler verebilirsin. veya css içerisinde daha genel tanımlamalar yapabilirsin.